features/org.wso2.carbon.core.common.feature/pom.xml

```diff
@@ -170,6 +168,11 @@
                                     org.wso2.carbon.crypto:org.wso2.carbon.crypto.provider:${carbon.crypto.version}
                                 </bundleDef>
                                 <bundleDef>org.wso2.orbit.sun.xml.bind:jaxb:${version.org.wso2.orbit.javax.xml.bind.jaxb}</bundleDef>
+                                <!-- TODO previously javax.activation package has come from org.eclipse.osgi_3.14.0.v20190517-1309 -->
+                                <bundleDef>org.wso2.orbit.javax.activation:activation:1.1.1.wso2v4</bundleDef>
+                                <!-- TODO previously javax.transaction package has come from org.eclipse.osgi_3.14.0.v20190517-1309 -->
```

Comment:
Temporarily removed the javax.transaction usages. Validate if this is actually required add a suitable dependency back.